src/compressor/Compressor.cc

```diff
@@ -54,6 +54,10 @@ const char *Compressor::get_comp_mode_name(int m) {
     case COMP_PASSIVE: return "passive";
     case COMP_AGGRESSIVE: return "aggressive";
     case COMP_FORCE: return "force";
+    case COMP_PASSIVE_LAZY: return "passive_lazy";
+    case COMP_AGGRESSIVE_LAZY: return "aggressive_lazy";
+    case COMP_FORCE_LAZY: return "force_lazy";
+
     default: return "???";
```

Comment:
I think we need following combination of compressions for regular operation and for scrub.

| regular-compression | scrub-compression |
| -------- | ------- |
| none  | none    |
| none | passive  |
| none | agres |
| none | force |
| passive | passive  |
| passive | agres  |
| passive | force |
| aggres | agres |
| aggres | force |
| force | force |

Reply:
oh... So you think we might need a separate lazy_compression_mode parameter (none/passive/aggressive/force) for background compression, right? Wouldn't that be an overkill for users?